### PR TITLE
Improves wizard spell descs, booleans

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
spellcheck: Changed a few wizard spell descriptions to better explain what they do.
/:cl:

I changed a few wizard spell descriptions, so the person rolling wizard for the first time in half a year has an idea of what the spells actually do.
Some had "Does not require wizard garb." at the end of their desc, which is pointless since `clothes_req = FALSE` does the same.

Also, replaced a bunch of 0/1 with booleans.

### CODE REVIEW

**Comment by 81Denton** [comment]
I wonder why my commit shows julianpicht as the author 🤔

**Comment by Kagus** [comment]
> I wonder why my commit shows julianpicht as the author 🤔

Welcome to the music industry, toots.

Also, woohoo for finally getting actual descriptions for Summon Lesser Guns/Arcane Barrage! An era of ignorance is not really at an end, but we can dream!

Should it be mentioned in the description of Rod Form that the power/distance of the spell also increases with spell investment, and not just the cooldown reduction? It's one of the rare spells that benefits in more ways than just getting a shorter downtime when you invest more points in it.

**Comment by 81Denton** [comment]
That's a good idea, let me add that.

**Comment by Kagus** [comment]
By the way, not only is julianpicht somehow listed as providing the commits for this PR, but McDoomBringer is also shown as having commented on it.

I think it's cursed.

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/spells/spell_types/wizard.dm [hunk 1]
```diff
@@ -370,7 +370,7 @@
 				M.electrocute_act(80, src, illusion = 1)
 		qdel(src)
 
-/obj/item/spellpacket/lightningbolt/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback)
```
Comment (AnturK) [inline code/modules/spells/spell_types/wizard.dm]: Would you mind updating arguments here to base ones, it's missing a few from what i can tell.

Comment (AnturK) [reply code/modules/spells/spell_types/wizard.dm]: Just add the arguments missing here from atom/movable define.

Comment (81Denton) [reply code/modules/spells/spell_types/wizard.dm]: Ahh okay. It looks like only `force = INFINITY` is missing.

Comment (81Denton) [reply code/modules/spells/spell_types/wizard.dm]: Couldn't figure out what you meant since I was looking at my local outdate branch heh

**Pull request merged**